soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -3276,12 +3276,16 @@ CustomMessage Randomizer::GetGoronMessage(u16 index) {
     return messageEntry;
 }
 
+std::string CUSTOM_ITEM_ICON(uint8_t customIconIndex, std::string msg) {
+    return char(customIconIndex) + msg;
+}
+
 void Randomizer::CreateCustomMessages() {
     // RANDTODO: Translate into french and german and replace GIMESSAGE_UNTRANSLATED
     // with GIMESSAGE(getItemID, itemID, english, german, french).
     const std::array<GetItemMessage, 85> getItemMessages = {{
-        GIMESSAGE(RG_GREG_RUPEE, ITEM_MASK_GORON, 
-			"You found %gGreg%w!",
+        GIMESSAGE(RG_GREG_RUPEE, ITEM_INVALID_4, 
+			CUSTOM_ITEM_ICON(0, "You found %gGreg%w!"),
```

Comment:
using magic numbers in here for now. once i know where things should live i'd like to move away from magic